Resources/ServerInfo/_NF/Guidebook/Rules/13_Command.xml

```diff
@@ -13,6 +13,9 @@ You are the highest authority in the sector apart from Central Command. You must
     -- Reopen your position at the station records computer.
     -- Return to cryosleep.
     -- If you cannot (disconnect, crash, power outage, etc.), ping the game admins in Discord.
+    -- Ensure you open your job slot prior to going to cryo.
+    -- You are welcome to open that slot early in order for a new SR or Sheriff to join so you can provide a briefing to them beforehand.
```

Comment:
```suggestion
    -- Opening a slot early in order to brief the next SR or Sheriff is recommended.
```
Might as well recommend it vs. saying "you can do this"

Reply:
That makes it sound more like a requirement and not an option.